user.css

```
@@ -83,34 +88,35 @@ body.spotui-spotify-enabled .Root__globalNav .main-globalNav-navLink,
+    color: var(--spotui-accent, #ff8c42) !important;
     border-radius: 0 !important;
 }
 
 body.spotui-spotify-enabled .Root__globalNav .main-globalNav-navLink:hover,
 body.spotui-spotify-enabled .Root__globalNav .main-globalNav-historyButtons button:hover,
 body.spotui-spotify-enabled .Root__globalNav .main-globalNav-searchContainer button:hover,
 body.spotui-spotify-enabled .Root__globalNav .main-globalNav-searchContainer [role="button"]:hover {
-    color: #ffb26f !important;
+    color: color-mix(in srgb, var(--spotui-accent, #ff8c42) 72%, white) !important;
+}
 }
```

Stylelint (17.14.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)